docs/sql/array.md

```diff
@@ -43,6 +43,7 @@ as `CREATE TABLE data AS (CITIES VARCHAR ARRAY, COUNTRY VARCHAR)`.
 | `ARRAY_JOIN`                                                         | Another name for `ARRAY_TO_STRING`                                                                                                                                                                                            |                                                                     |
 | `ARRAY_LENGTH(` _array_ `)`                                          | Another name for `CARDINALITY`                                                                                                                                                                                                | `ARRAY_LENGTH( ARRAY [1, 2, 3])` => `3`                             |
 | `ARRAY_POSITION(` _array_ `, ` _element_ `)`                         | Returns the (1-based) index of the first _element_ of the _array_ as a long. Returns 0 if _element_ doesn't exist in _array_. If the _element_ is `NULL`, `NULL` is returned.                                                 | `ARRAY_POSITION(ARRAY [1, 3, 4, 6], 4)` => 3                        |
+| `ARRAY_REPEAT(` _element_ `, ` _count_ `)`                           | Returns the array containing _element_ _count_ times. If _count_ is negative, an empty array is returned. If _element_ is `NULL`, an array containing _count_ number of `NULL`s is returned.                                  | `ARRAY_REPEAT(3, 4)` => `[3, 3, 3, 3]`                              | 
```

Comment:
Didn't mention the case for count being NULL as that gives an error right now. https://github.com/feldera/feldera/issues/1473

Reply:
I think we should document the expected behavior and keep the issue open to indicate that we have a bug.
Otherwise we may forget to fix the documentation when we fix the bug.
But your proposal is also reasonable.